### DISCUSSION TITLE
Maandag 02-12-24

### DISCUSSION THREAD

**Discussion by Nazneen05x** [rutgerkock/buurtcampus-oost#14] [Stand-ups]
- Sprint doel (wat wil je af hebben deze sprint?)
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by wingsvn** [comment]
**Sprint doel (wat wil je af hebben deze sprint?)**
De rest van de agenda pagina, en de (een aantal) should haves van de zaden pagina.

**Hoe gaat het met je en waar ben je nú mee bezig?**
Het gaat oke. ik ben bezig met de deeltaak creative coding.

**Wat wil je deze week bereiken (aan werken)?**
Ik denk dat ik deze week voornamelijk bezig zal zijn met de deeltaak.

**Waar loop je tegen aan?**
Tijd/planning. Aangezien we bezig zijn met een nieuwe deeltaak waar we iets "geks" moeten maken en dit niet echt aansluit met de huisstijl, is er minder tijd over voor de bieb-in-bloei website zelf.

**Comment by Nazneen05x** [comment]
**Sprint doel (wat wil je af hebben deze sprint?)**
Deze sprint wil ik het menu en de nieuwe over pagina af hebben. 

**Hoe gaat het met je en waar ben je nú mee bezig?**
Het gaat wel. Ik ben vorige sprint druk bezig geweest met het opnieuw bouwen van het menu. Ik moet nog een paar bugs oplossen en dan is het menu klaar!

**Wat wil je deze week bereiken (aan werken)?**
Deze week wil ik beginnen met de nieuwe over pagina bouwen en hopelijk ook gelijk deze week afronden.

**Waar loop je tegen aan?**
Op dit moment alleen bugs van het menu.

**Comment by Annevd** [comment]
**Sprint doel (wat wil je af hebben deze sprint?)**
Ik zou graag alle must haves af willen hebben van ten minste de contact pagina, en een begin maken aan de zaden pagina.

**Hoe gaat het met je en waar ben je nú mee bezig?**
Het gaat wel goed, ik ben nu bezig met de deeltaak creative coding

**Wat wil je deze week bereiken (aan werken)?**
Ook ik zal deze week vooral aan de deeltaak werken. Mocht ik eerder klaar zijn zal ik kijken naar het projectplan.

**Waar loop je tegen aan?**
Het is lastig om het projectplan bij te houden terwijl we ook veel moeten doen aan creative coding.

**Comment by Lmikkers** [comment]
### Sprint doel (wat wil je af hebben deze sprint?)
Ik zou graag Harry af willen ronden en alle must haves van mijn planning

### Hoe gaat het met je en waar ben je nú mee bezig?
Ik ben bezig met de deeltaak creative coding, nu met schetsen maken van wat ik wil maken. Ook ben ik bezig met mijn Midterm herkansing van vorig jaar deze moet ik vrijdag inleveren. 

### Wat wil je deze week bereiken (aan werken)?
Ik wil deze week werken aan mijn deeltaak.

### Waar loop je tegen aan?
Ik vergeet vaak de standup in te vullen... dit moet ik gelijk gaan doen. Ook vind ik het lastig om bezig te zijn voor bieb in bloei, vergeet dit soms omdat ik het dan druk heb met schoolopdrachten
